sources/api/certdog/src/main.rs

```diff
@@ -246,6 +249,20 @@ mod error {
             source: Box<apiclient::Error>,
```

Comment:
I can't comment on line 245: But the `APIRequest` error is no longer used. One quirk of Snafu's macros is that the compiler can't detect unused Error variants because Snafu is generating code in which they are used. Same with the `ResponseJson` Error variant.